src/components/CheckboxLegacy/CheckboxLegacy.test.js

```diff
@@ -8,40 +8,44 @@ import CheckboxLegacy from 'components/CheckboxLegacy';
 describe('CheckboxLegacy component', () => {
   test('should render correctly without props', () => {
     render(<CheckboxLegacy />);
-    expect(screen.getByRole('checkbox')).toBeInTheDocument();
+    expect(screen.getByTestId('checkbox-legacy')).toBeInTheDocument();
```

Comment:
to expand on my comment from the previous file, if we had 2 checkboxes rendered, the tests here would fail :) that's why i think it's important to make them unique